plugins/node/instrumentation-react-native-navigation/package.json

```
@@ -0,0 +1,79 @@
+{
+  "name": "@opentelemetry/instrumentation-react-native-navigation",
+  "version": "0.1.0",
+  "description": "OpenTelemetry instrumentation for `@react-navigation/native`, `expo-router` and `wix/react-native-navigation` route changes for React Native applications",
+  "keywords": [
+    "navigation",
+    "react-native",
+    "instrumentation",
+    "nodejs",
+    "opentelemetry",
+    "profiling",
+    "tracing"
+  ],
+  "homepage": "https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/instrumentation-react-native-navigation#readme",
+  "license": "Apache-2.0",
+  "author": "OpenTelemetry Authors",
+  "bugs": {
+    "url": "https://github.com/open-telemetry/opentelemetry-js-contrib/issues"
+  },
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "clean": "rimraf build/*",
+    "compile": "tsc -p .",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "lint:readme": "node ../../../scripts/lint-readme.js",
+    "precompile": "tsc --version && lerna run version:update --scope @opentelemetry/instrumentation-react-native-navigation --include-dependencies",
+    "prewatch": "npm run precompile",
+    "prepublishOnly": "npm run compile",
+    "test": "nyc mocha",
+    "test-all-versions": "tav",
+    "version:update": "node ../../../scripts/version-update.js",
+    "watch": "tsc -w",
+    "postinstall": "rm -rf ./node_modules/react"
+  },
+  "peerDependencies": {
+    "@opentelemetry/api": "^1.3.0",
+    "react-dom": "18.0.0"
```

I think we should just enforce a minimum here since it's a peer dependency, otherwise it locks the user to that specific version in their application:

```suggestion
    "react-dom": ">=18.0.0"
```